### PR TITLE
Add documentation: training a model using a custom dataset

### PR DESCRIPTION
**Pull request by JungeAlexander**
The following sections should be added/updated in `doc/example/example.md`:

# Data preparation

- outline tab-delimited format: pmid, # paragraph, # sentence, disease, gene, text
- point out how this generalizes beyond disease-gene pairs and single-pairs

## Blanking entity names

- replace with wildcards in text
- point to tokens used in example dataset

# Using a pre-trained scoring model

[... existing part on using pre-trained model goes here...]

# Training and applying a custom scoring model

## Label column

- only binary labels as last column of data tsv 
- add a command to get text into correct format for training

## Evaluation of training and test set performance

### CODE REVIEW

**Comment by JungeAlexander** [comment]
[doc custom data](https://trello.com/c/Dut02EPm/1181-doc-custom-data)
